ui/app/components/app/account-menu/index.scss

```
@@ -53,8 +53,13 @@
 
   &__accounts-container {
     display: flex;
+    position: relative;
     flex-direction: column;
     z-index: 200;
+
+    @media screen and (max-width: 575px) {
```

Why `screen` and `max-width`? This is a height issue. And why `575px`? The popup has a height of 600px, but it looks like the menu is cut off until it gets above ~625px or so

```suggestion
    @media (max-height: 625px) {
```

---

I just followed the other media queries in the file, all of which seem to be about handling the popup UI, and all of which use `@media screen`, and then `max-width` or `min-width`. See: https://github.com/MetaMask/metamask-extension/blob/develop/ui/app/components/app/account-menu/index.scss

---

In fact, there are 177 existing media queries in our code base, and ~175 of them use `@media screen and [max-|min-]width`.

---

I'm not sure what that has to do with this situation; I don't know what each of those are intended for. The other examples in this file of setting `max-height` in a `max-width` media query block also seem strange to me, but it's hard to say without knowing what they're intended to do.

---

imo we'd target height here in the media query. It seems like having a discussion on media query formats and pulling those queries into some variables in SCSS would be a good idea in the long term.

---

Very well; although I think we can pretty safely say that the purpose of those media queries is to apply styles to the popup UI, since their effect is to apply styles to the popup UI, and we'd expect to see such queries all over the codebase.

Whatever the case, we can address it later like Brad suggested.

Changed to `max-height: 600px` query in b98e0a9, since 600px is the height of the popup UI. It works.

---

This is certainly an improvement, but it still leaves the `Settings` menu option obscured when the window is between `600px` and `625px`, as uncommon as those heights may be.

If we would prefer keeping the media queries restricted to certain screen sizes to simplify testing, we could pursue alternate solutions like setting a `max-height` based upon the viewport height.

---

Ah, now I see what you mean. Sorry about that. I'll create an issue attempting to document these concerns. Ref: #8711